docgen/src/Styling.md

```diff
@@ -10,58 +10,133 @@ All widgets that render one or more DOM nodes accept a `theme` prop. This prop i
 The different theme keys supported by components are described on their respective documentation page. You can also provide themes using css-modules, Aphrodite, JSS and others if you want. 
 See [`react-themeable`](https://github.com/markdalgleish/react-themeable) for the underlying implementation and options.
 
-By default, we provide `defaultTheme` for each widgets using css-modules under the hood. Most widgets have empty default theme, but the `SearchBox`, `RangeRatings`, `Range` and `Pagination` widgets 
+By default, we provide `defaultClassNames` for each widgets using css-modules under the hood. Most widgets have raw style but the `SearchBox`, `RangeRatings`, `Range` and `Pagination` widgets 
 are delivered with a built-in one and are ready to use. 
 
-You can access every default theme easily. Here is an example if you want to retrieve the `SearchBox` default theme:
+In order to get you started with styling, we are going to use the example you made in the [Getting Started Tutorial](gettingstarted.html) and add some style to it. 
+If you haven't follow the [Getting Started Tutorial](gettingstarted.html) yet, please do and comeback here right after. 
 
-```
-const searchBoxDefaultTheme = SearchBox.defaultTheme;
-```
+In this tutorial, you'll learn how to: 
 
-### Example: Styling a SearchBox
+* Customize a default provided theme. 
+* Create a totally new theme for a widget.  
 
-#### Redefining class names 
+Before we start, you need to know that this tutorial will use css-modules as a way to style components. Be aware that you are totally free to use any other way supported by [`react-themeable`](https://github.com/markdalgleish/react-themeable). 
+Please also note that every `react-instantsearch` widgets already have class names and if you want you can use them directly. 
 
-If you want to use pure css on your side, either you can use the generated class names we provide for each widgets, or you can redefine your own. You can also directly use some css. 
+## Modify the SearchBox colors 
 
+The `SearchBox` is the most opinionated widget that you will find in `react-instantsearch`. It's already nice out-of-the-box but the submit button background color may not fit your own theme.
+To change it, let's create a new css file called `searchBox.css`. 
+
+```css
+.submit {
```

Comment:
Maybe we can explain how did we get the info that we needed to change `.submit`? Can I do that using the css files in the project? Or should I do that from the chrome dev tools?